src/esn.calendar.libs/app/app.config.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+angular.module('esn.calendar.libs')
+  .config(registerI18N);
+
+function registerI18N($translateProvider) {
+  $translateProvider.translations('en', require('../../linagora.esn.calendar/i18n/en.json'));
```

Comment:
this module should have its own i18n files. like `en.json` but with only the needed translations for `esn.calendar.libs`